sql/updates/reverse-dev.sql

```diff
@@ -437,3 +437,11 @@ ALTER TABLE _timescaledb_catalog.hypertable_data_node
     ADD CONSTRAINT hypertable_data_node_hypertable_id_fkey FOREIGN KEY (hypertable_id) REFERENCES _timescaledb_catalog.hypertable(id);
 ALTER TABLE _timescaledb_catalog.tablespace
     ADD CONSTRAINT tablespace_hypertable_id_fkey FOREIGN KEY (hypertable_id) REFERENCES _timescaledb_catalog.hypertable(id) ON DELETE CASCADE;
+
+DROP FUNCTION @extschema@.ByHash(name, integer, regproc);
```

Comment:
Is the Pascal case notation of `ByHash` and `ByRange` intentional?

Reply:
It was, but I have no problems changing names if you have better proposals.

Reply:
Here are some alternative names that was used in early versions:
- `HashPartition` and `RangePartition`. They are quite long though.
- `HashPart` and `RangePart`. A little to terse to be understandable.
- `by_hash` and `by_range`. Just an alternative if snake case is preferred.

Reply:
I have no strong opinion here. Just wanted to understand if this is intentional. Personally, I would prefer `by_range` and `by_range`. This looks more like our existing API functions.

Reply:
`by_range` and `by_hash` it is.